### PR TITLE
Feature/vxlan fix

### PR DESCRIPTION
**Pull request by anramgop**

### CODE REVIEW

File: tests/ciscotest.rb [hunk 1]
```diff
@@ -241,13 +241,35 @@ def mt_full_interface?
...
+    # Example 'show mod' output to match against:
+    #   '9  12  10/40 Gbps Ethernet Module  N7K-F312FQ-25 ok'
+    #   '9  12  10/40 Gbps Ethernet Module  N77-F312FQ-25 ok'
+    #   '2   6  Nexus 6xQSFP Ethernet Module  N5K-C5672UP-M6Q ok'
+    #   '2   6  Nexus xxQSFP Ethernet Module  N6K-C6004-96Q/EF ok'
+    if node.product_id[/N(5|6)K/]
+      sh_mod_string = @device.cmd("sh mod | i '^[0-9]+.*N[56]K-C[56]'")
+      sh_mod = sh_mod_string[/^(\d+)\s.*N[56]K-C(56|6004)/]
+      slot = sh_mod.nil? ? nil : Regexp.last_match[1]
+      skip('Unable to find compatible interface in chassis') if slot.nil?
```
Comment (robert-w-gries) [inline tests/ciscotest.rb]: The unnecessary line is back again.  Looks like your original removal was hiding in the merge commit.

``` diff
-      slot = sh_mod.nil? ? nil : Regexp.last_match[1]
-      skip('Unable to find compatible interface in chassis') if slot.nil?
+      skip('Unable to find compatible interface in chassis') if sh_mod.nil?
```

File: tests/test_vxlan_vtep.rb [hunk 2]
```diff
@@ -30,6 +30,9 @@ def setup
   end
 
   def teardown
+    Feature.nv_overlay_disable
```
Comment (robert-w-gries) [inline tests/test_vxlan_vtep.rb:33]: Seeing errors on `n56-136` due to the feature disable:

```
  1) Error:
TestVxlanVtep#test_description:
Cisco::CliError: [Cisco::Feature] The command 'no feature nv overlay' was rejected with error:
{}
```

Comment (robert-w-gries) [inline tests/test_vxlan_vtep.rb:33]: More helpful message:

```
Error: no feature nv overlay failed enabling/disabling feature is in progress
```

Comment (anramgop) [inline tests/test_vxlan_vtep.rb:33]: NVE and fabricpath are enabled. Looks like n56-136 is in a failed state - will need a reboot. This is my run on my 5k:

ruby tests/test_vxlan_vtep.rb -e n6k -v
Run options: -e n6k -v --seed 5065

Running:

Node under test:
- name  - n6k-3
- type  - N6K-C6001-72UP
- image - bootflash:///aneesh-s.bin

TestVxlanVtep#test_shutdown = 17.09 s = .
TestVxlanVtep#test_create_negative = 16.29 s = .
TestVxlanVtep#test_source_interface_hold_down_time = 15.67 s = .
TestVxlanVtep#test_mt_full_create_destroy_multiple = 2.68 s = S
TestVxlanVtep#test_source_interface = 22.05 s = .
TestVxlanVtep#test_description = 17.71 s = .
TestVxlanVtep#test_create_destroy_one = 17.18 s = .
TestVxlanVtep#test_host_reachability = 17.62 s = .

Finished in 128.535827s, 0.0622 runs/s, 0.1945 assertions/s.

  1) Skipped:
TestVxlanVtep#test_mt_full_create_destroy_multiple [tests/test_vxlan_vtep.rb:86]:

**Comment by robert-w-gries** [comment]
👍

**Pull request merged**